test/src/testFixtures/groovy/org/openremote/test/ContainerTrait.groovy

```diff
@@ -77,6 +78,10 @@ import static org.openremote.model.Constants.MASTER_REALM_ADMIN_USER
 
 trait ContainerTrait {
 
+    static {
+        LogUtil.initialiseJUL()
+    }
```

Comment:
Double checking whether this was for testing purposes or not, since there were no other logging changes.

Reply:
Yes needed to initialise logging in tests